GitUI/FormProcess.cs

```diff
@@ -132,7 +132,7 @@ protected virtual void BeforeProcessStart()
         {
         }
 
-        private void processStart(FormStatus form)
+        private new void ProcessStart(FormStatus form)
```

Comment:
❓ Why is `new` required here? It would be nice to avoid this, e.g. by choosing a different name.

Reply:
agree

Reply:
Base class defined nested delegates with these names. I've exchanged them with `Action<FormStatus>` and cleaned up in 9e92993f2.